docs/docs/themes/using-multiple-gatsby-themes.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Using Multiple Gatsby Themes
+título: Usando de múltiples temas de Gatsby
 ---
 
-Gatsby themes are intended to be composable. This means you can install multiple themes alongside each other.
+Gatsby temas están destinados a ser componible. Esto significa que puede instalar temas múltiples el uno junto al otro.
```

Comment:
```suggestion
Gatsby temas están destinados a ser combinados. Esto significa que puedes instalar varios temas el uno junto al otro.
```

Reply:
Gatsby temas están destinados a ser combinados. => Los temas Gatsby están destinados a ser combinados.

Reply:
Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.